.ci/cico_updates_minishift.sh

```diff
@@ -102,47 +105,50 @@ waitForNewCheVersion() {
   fi
 }
 
-self_signed_minishift() {
-  export DOMAIN=*.$(minishift ip).nip.io
-
-  source ${OPERATOR_REPO}/.ci/util/che-cert-generation.sh
-
-  #Configure Router with generated certificate:
-
-  oc login -u system:admin --insecure-skip-tls-verify=true
-  oc project default
-  oc delete secret router-certs
-
-  cat domain.crt domain.key > minishift.crt
-  oc create secret tls router-certs --key=domain.key --cert=minishift.crt
-  oc rollout latest router
-
-  oc create namespace che
-
-  cp rootCA.crt ca.crt
-  oc create secret generic self-signed-certificate --from-file=ca.crt -n=che
+# Utility to get che events and pod logs from openshift
+function getOCCheClusterLogs() {
+  mkdir -p /tmp/artifacts-che
+  cd /tmp/artifacts-che
+
+  for POD in $(oc get pods -o name -n ${NAMESPACE}); do
+    for CONTAINER in $(oc get -n ${NAMESPACE} ${POD} -o jsonpath="{.spec.containers[*].name}"); do
+      echo ""
+      echo "[INFO] Getting logs from $POD"
+      echo ""
+      oc logs ${POD} -c ${CONTAINER} -n ${NAMESPACE} |tee $(echo ${POD}-${CONTAINER}.log | sed 's|pod/||g')
+    done
+  done
+  echo "[INFO] Get events"
+  oc get events -n ${NAMESPACE}| tee get_events.log
+  oc get all | tee get_all.log
 }
 
-testUpdates() {
+function minishiftUpdates() {
   # Install previous stable version of Eclipse Che
-  self_signed_minishift
   installLatestCheStable
 
   # Create an workspace
   getCheAcessToken # Function from ./util/ci_common.sh
   chectl workspace:create --devfile=${OPERATOR_REPO}/.ci/util/devfile-test.yaml
 
   # Change operator images defaults in the deployment
-  sed -i -e "s|${previousPackageVersion}|${lastPackageVersion}|" "${OPERATOR_REPO}/tmp/che-operator/operator.yaml"
+  sed -i'.bak' -e "s|${previousPackageVersion}|${lastPackageVersion}|" "${OPERATOR_REPO}/tmp/che-operator/operator.yaml"
```

Comment:
Sorry, what does it mean  `-i'.bak'` ?

Reply:
Creates a backup of your original file. In MacOS for some reason is mandatory in this cases